crates/forge-runner/src/coverage_api.rs

```diff
@@ -12,6 +13,11 @@ pub fn run_coverage(saved_trace_data_paths: &[PathBuf]) -> Result<()> {
         .ok()
         .unwrap_or_else(|| PathBuf::from("cairo-coverage"));
 
+    ensure!(
+        which(coverage.as_os_str()).is_ok(),
+        "The 'cairo-coverage' binary was not found in PATH. Perhaps you didn't install it? Check out docs for more info"
```

Comment:
nit:
```suggestion
        "The 'cairo-coverage' binary was not found in PATH. It may not have been installed. Please refer to the documentation for installation instructions."
```

Reply:
I like the suggestions, but how about we also show a relevant link instead of telling the user to check the documentation?

Reply:
I was thinking about that, but I have a bad feeling that nobody will update this link if anything changes.

Reply:
How about linking to the coverage repository at least? This is very unlikely to change.